src/gazebo_camera_manager_plugin.cpp

```diff
@@ -540,6 +545,55 @@ void CameraManagerPlugin::_handle_stop_take_photo(const mavlink_message_t *pMsg,
                   MAV_CMD_IMAGE_STOP_CAPTURE, MAV_RESULT_ACCEPTED, srcaddr);
 }
 
+void CameraManagerPlugin::_handle_start_video_capture(const mavlink_message_t *pMsg, struct sockaddr* srcaddr)
+{
+    gzdbg << "Handle Start Video Capture" << endl;
+
+    mavlink_command_long_t cmd;
+    mavlink_msg_command_long_decode(pMsg, &cmd);
+    std::lock_guard<std::mutex> guard(_captureMutex);
+
+    if (_captureMode != CAPTURE_DISABLED) {
+        // We are already capturing
+        _send_cmd_ack(pMsg->sysid, pMsg->compid, MAV_CMD_VIDEO_START_CAPTURE, MAV_RESULT_TEMPORARILY_REJECTED, srcaddr);
+        return;
+    }
+
+    if (cmd.param1 != 0 || cmd.param2 != 0) {
+        gzerr << "VIDEO_START_CAPTURE: param1 and param2 must be 0\n";
+        _send_cmd_ack(pMsg->sysid, pMsg->compid, MAV_CMD_VIDEO_START_CAPTURE, MAV_RESULT_UNSUPPORTED, srcaddr);
```

Comment:
[MAV_RESULT_UNSUPPORTED](https://mavlink.io/en/messages/common.html#MAV_RESULT_UNSUPPORTED). is to be used when the whole command is not supported, I think.

In this case I would say [MAV_RESULT_DENIED](https://mavlink.io/en/messages/common.html#MAV_RESULT_DENIED).

Reply:
Ah, was duplicating how _handle_take_photo handles basically the same thing. Should I change that code as well to MAV_RESULT_DENIED?

Reply:
You can see it here: https://github.com/PX4/PX4-SITL_gazebo-classic/blob/main/src/gazebo_camera_manager_plugin.cpp#L526